alto/commands/cromwell/list_jobs.py

```diff
@@ -0,0 +1,164 @@
+import argparse
+import getpass
+import requests
+import pandas as pd
+from datetime import datetime
+from dateutil import parser
+import time
+
+
+def datetime_from_utc_to_local(utc_datetime: str) -> str:
+    if not utc_datetime:
+        return ""
+    now_timestamp = time.time()
+    offset = datetime.fromtimestamp(now_timestamp) - datetime.utcfromtimestamp(
+        now_timestamp
+    )
+    return (parser.parse(utc_datetime) + offset).strftime("%d %b %Y, %H:%M:%S")
+
+
+class bcolors:
+    SUCCESS = "\033[92m"
+    FAILED = "\033[91m"
+    ENDC = "\033[0m"
+
+
+def show_one_job(s: str, status: str):
+    if status == "Succeeded":
+        print(f"{bcolors.SUCCESS}{s}{bcolors.ENDC}")
+    elif status in ["Failed", "Aborted"]:
+        print(f"{bcolors.FAILED}{s}{bcolors.ENDC}")
+    else:
+        print(s)
+
+
+def show_jobs(df: pd.DataFrame):
+    if "creator" not in df.columns:
+        df["creator"] = ""
+    print(
+        "{:<38} {:<16} {:<24} {:<13} {:<28} {:<28} {:<28}".format(
+            "Job ID", "Creator", "Workflow", "Status", "Submitted", "Start", "End"
+        )
+    )
+
+    for _, row in df.iterrows():
+        show_str = "{:<38} {:<16} {:<24} {:<13} {:<28} {:<28} {:<28}".format(
+            row["id"],
+            row["creator"],
+            row["name"],
+            row["status"],
+            row["submission"],
+            row["start"],
+            row["end"],
+        )
+        show_one_job(show_str, row["status"])
+
+
+def list_jobs(
+    server: str, port: int, is_all: bool, username: str, job_id: str, job_status: str
```

Comment:
`job_status` should have type signature `Optional[str]`, as its default is `None` due to `main` function.